docs/_docs/windows.md

```diff
@@ -42,16 +43,24 @@ Now all that is left to do is install Jekyll.
 sudo gem install jekyll bundler
 ```
 
-You can test by running:
+Check to see if your installation worked.
 
 ```
-jekyll new my_project
+jekyll -v
 ```
 
 **And that's it!**
+
+To start a new project, just run:
+```
+jekyll new my_project
+```
+
+replacing `my_project` with the name of your website.
+
 If you `cd` into the folder, you can make sure time management is working by opening your `_posts` folder. You should see a markdown file with the current date listed.
```

Comment:
instead of broken sentences, you could write it as:

---
To start a new project named `my_project` simply run:
```
$ jekyll new my_product
```
---